evennia/utils/evtable.py

```diff
@@ -199,6 +199,62 @@ def _split(self, text):
 
         return chunks
 
+    def _handle_long_word(self, reversed_chunks, cur_line, cur_len, width):
+        """_handle_long_word(chunks : [string],
+                             cur_line : [string],
+                             cur_len : int, width : int)
+
+        Handle a chunk of text (most likely a word, not whitespace) that
+        is too long to fit in any line.
+        """
```

Comment:
Please use Google doc style. See Evennia's style guide and all other methods in this module.